package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hot-formula-parser",
-  "version": "4.3.0-unc7",
+  "version": "4.3.1-unc8",
```

Comment:
bumping the minor version here to see if that fixes the package lock issue we have when updating from this repo in the main app